ember-cli-build.js

```diff
@@ -2,13 +2,26 @@
 
 const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 
+const webpackConfig = {
```

Comment:
It seems ember-auto-import needs the same fix, we just didn't notice because we don't run a non-embroider scenario without a lock file.